src/Sarif.Viewer.VisualStudio/ErrorList/SarifResultTableEntry.cs

```diff
@@ -79,9 +107,7 @@ public SarifResultTableEntry(SarifErrorListItem error)
             this.columnKeyToContent[StandardTableKeyNames.ProjectName] = this.Error.ProjectName;
 
             string supressionState = this.Error.VSSuppressionState.ToString();
```

Comment:
should we create a fixed variable containing the strings instead of creating the string each time that this method is called?

Reply:
No, it varies from input `error` to input `error` (note that `this.Error = error`.

---
In reply to: [538282371](https://github.com/microsoft/sarif-visualstudio-extension/pull/313#discussion_r538282371) [](ancestors = 538282371)